java/fabric/egg-fabric.json

```
@@ -1,63 +1,59 @@
 {
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PANEL",
     "meta": {
-        "version": "PTDL_v2",
+        "version": "PLCN_v3",
         "update_url": null
     },
-    "exported_at": "2024-08-02T12:14:52+00:00",
+    "exported_at": "2025-11-08T21:39:55+00:00",
     "name": "Fabric",
     "author": "accounts@bofanodes.io",
     "uuid": "78b02ebb-fec8-49c5-943c-ca4aa117b693",
     "description": "Fabric is a modular modding toolchain targeting Minecraft 1.14 and above, including snapshots.",
+    "image": null,
+    "tags": [],
     "features": [
         "eula",
         "java_version",
         "pid_limit"
     ],
     "docker_images": {
-        "Java 8": "ghcr.io\/parkervcp\/yolks:java_8",
-        "Java 11": "ghcr.io\/parkervcp\/yolks:java_11",
-        "Java 16": "ghcr.io\/parkervcp\/yolks:java_16",
-        "Java 17": "ghcr.io\/parkervcp\/yolks:java_17",
-        "Java 21": "ghcr.io\/parkervcp\/yolks:java_21"
+        "Java 8": "ghcr.io/parkervcp/yolks:java_8",
+        "Java 11": "ghcr.io/parkervcp/yolks:java_11",
+        "Java 16": "ghcr.io/parkervcp/yolks:java_16",
+        "Java 17": "ghcr.io/parkervcp/yolks:java_17",
+        "Java 21": "ghcr.io/parkervcp/yolks:java_21"
```

change the yolks repository to pelican-eggs and add java 21
```suggestion
        "Java 8": "ghcr.io/pelican-eggs/yolks:java_8",
        "Java 11": "ghcr.io/pelican-eggs/yolks:java_11",
        "Java 16": "ghcr.io/pelican-eggs/yolks:java_16",
        "Java 17": "ghcr.io/pelican-eggs/yolks:java_17",
        "Java 21": "ghcr.io/pelican-eggs/yolks:java_21",
        "Java 25": "ghcr.io/pelican-eggs/yolks:java_25"
```